code/modules/asset_cache/asset_list.dm

```diff
@@ -361,6 +361,7 @@ GLOBAL_LIST_EMPTY(asset_datums)
 		queuedInsert(sprite_name, I, icon_state, dir, frame, moving)
 	else
 		to_generate += list(args.Copy())
```

Comment:
Bigger asset init savings, please confirm sleeping is okay here.

Reply:
It isn't, from experience

Reply:
Changed this particular case